packages/bpk-component-datepicker/src/BpkDatepicker.tsx

```diff
@@ -381,12 +380,10 @@ class BpkDatepicker extends Component<Props, State> {
             <BpkPopover
               id={`${id}-popover`}
               target={input}
-              renderTarget={renderTarget}
```

Comment:
I guess this prop would have to be deprecated from `BpkDatepicker` once we migrate the Modal as well? 🤔

Reply:
Good question, will take a look at this!

Reply:
Yes you are right this prop will begin to go away when we pick up the modal ticket!